spec/features/state_file/complete_intake_spec.rb

```diff
@@ -732,7 +732,7 @@
 
       groups = page.all(:css, '.white-group').count
       h2s = page.all(:css, 'h2').count
-      expect(groups).to eq(h2s)
+      expect(groups).to eq(h2s - 1) # account for the h2 reveal button
```

Comment:
[DUST] is there a test we can add here to validate that the detailed review section contains all the different fields we want it to?